clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1223,6 +1233,26 @@ public void maybeAutoCommitOffsetsAsync(long now) {
         }
     }
 
+    private boolean waitForPendingAsyncCommits(Timer timer) {
```

Comment:
can we name this: `invokePendingAsyncCommits` ?
so that we could do:
```
if (!invokePendingAsyncCommits(timer) { return false; }
equestFuture<Void> future = sendOffsetCommitRequest(...)
```

Reply:
or maybe 
```
if (offset.isEmpty() && inflightCommit.get() == 0) return true;
... do the rest
```

Reply:
Renamed the method in 1fa48f53f0e6f5a2a9821075fa053e01cba6b0b2.

Also, see this comment: https://github.com/apache/kafka/pull/13678#discussion_r1190730213